pkg/oc/cli/admin/release/new.go

```diff
@@ -1466,6 +1475,32 @@ func pruneEmptyDirectories(dir string) error {
 	})
 }
 
+// ensureConsistentSources checks that images built from the same Git repository are from the same Git commit.
+func ensureConsistentSources(is *imageapi.ImageStream) error {
```

Comment:
nit: this function should return the error with all the different commits for a location, not the difference to first found....

Reply:
> nit: this function should return the error with all the different commits for a location...

Dup of [your other comment][1]?  I'll mark this resolved based on the change I referenced there, but please re-open if there is another facet of this that I'm overlooking.

[1]: https://github.com/openshift/origin/pull/22816#discussion_r285365489

Reply:
For example today MCO has 4 images.

When all 4 differ error will be something like
mcc was xx mco was x for mco repo
mcs was xy mco was x for mco repo
mcd was xz mco was x for mco repo.


But I guess a better error message would be
mco repo has inconsistent commits mco x mcc xx mcs xy mcd xz

Reply:
> When all 4 differ error will be something like...

Yeah, I'm fine with that.  Doesn't seem like something that is going to come up often, and we can always come back and polish the error message if it does.